wireapp/wire-android-reloaded
# feat: add process flow after asset message download [AR-1323]

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

After downloading an asset message into the internal storage of the app, we wanted to show a dialog to ask the user how she wanted to consume the recently downloaded asset, either `open` it directly with any suitable app or to simply `download` the asset to the external `Downloads` folder of the device.

### Solutions

There will be 

### Dependencies (Optional)

This PR should not be merged before its [Kalium counterpart](https://github.com/wireapp/kalium/pull/496) gets merged too.

Needs releases with:


### Testing
Coming in soon
#### Test Coverage (Optional)


#### How to Test

Just open any conversation, send or receive an asset file, click on the asset message and check both of the options of the pop up.

### Attachments (Optional)

https://user-images.githubusercontent.com/2468164/168305112-ba49d18e-1ca0-42fd-ad57-f3a27d2a81d7.mov

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
